test/Microsoft.ML.Tests/Transformers/CustomTransformerTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Data.IO;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Runtime.RunTests;
+using Microsoft.ML.Runtime.Tools;
+using System.ComponentModel.Composition;
+using System.ComponentModel.Composition.Hosting;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.Tests.Transformers
+{
+    public sealed class CustomTransformerTests : TestDataPipeBase
+    {
+        public CustomTransformerTests(ITestOutputHelper helper) : base(helper)
+        {
+        }
+
+        public class MyInput
+        {
+            public float Float1 { get; set; }
+            public float[] Float4 { get; set; }
+        }
+
+        public class MyOutput
+        {
+            public string Together { get; set; }
+        }
+
+        public class MyLambda
+        {
+            [Export("MyLambda")]
+            public ITransformer MyTransformer => new CustomMappingTransformer<MyInput, MyOutput>(ML, MyAction, "MyLambda");
+
+            [Import]
+            public MLContext ML { get; set; }
+
+            public static void MyAction(MyInput input, MyOutput output)
+            {
+                output.Together = $"{input.Float1} + {string.Join(", ", input.Float4)}";
```

Comment:
Is there a test that asserts this gets executed correctly? I don't see a test that asserts the output string is the expected value.

Reply:
Modified to assert it.

---
In reply to: [229416231](https://github.com/dotnet/machinelearning/pull/1406#discussion_r229416231) [](ancestors = 229416231)